skillful_network_client/src/app/home/news-feed/list-publications/list-publications.component.html

```diff
@@ -0,0 +1 @@
+
```

Comment:
Why pushing list-publications when it is not done yet?
You could have a smaller pull-request by pushing only the files about adding publications, right?